_sass/scaffolds.scss

```diff
@@ -370,7 +386,11 @@ li.sticky-nav{
 
 @media (min-width: 1281px) {
   body, p, ol, ul, td, pre {
-    font-size: 22px;
+    font-size: 20px;
     line-height: 25px;
   }
 }
+
+#page-sticky-nav {
+  background: transparent;
```

Comment:
does this need to be here? what is the intention of this?

Reply:
this was because of the 'sticky-nav' titles in the 'archives' layout pages covering the linez. not sure if it's still necessary but that was the intention.

Reply:
not necessary anymore, pullin it!

Reply:
ah, gotcha! thanks for explaining. yeah, `sticky` thingz really need opaque background because content flows behind them.